### ISSUE TITLE
Compiler warning in DataRepresentable.swift

### ISSUE THREAD

**Issue by duskoo** [Co-Epi/app-ios#231]
> Initialization of 'UnsafeBufferPointer<Self>' results in a dangling buffer pointer

The file seems unused and safe to remove (remnant from before the inclusion of The TCNClient lib).

**Comment by ivnsch** [comment]
Yeah, we actually have to do a cleanup, removing all CEN and pre-Rust leftovers (which would include renaming things with "CEN" that are still needed to "TCN"). I was waiting after beta testing of 0.3 (to be more sure that we will keep the Rust lib) before doing this.
